backend/curriculum_tracking/tests/for_testing_purposes.py

```diff
@@ -0,0 +1,278 @@
+import unittest
```

Comment:
I should actually just add the file to .gitignore because it's not actually what I want at the the end of the day, it's just something I use to fool around with and check that stuff works as I expect them to work.  That's why it doesn't have very descriptive name seeing that I am going to delete it anyway.

Reply:
ok cool.

There's a trick I use. If you make a directory named `gitignore` then you  can put your rough work files in there and they'll get ignored. That way you dont need to add each file individualy